src/tutorials/pet-shop.md

```diff
@@ -375,6 +375,99 @@ Since arrays can only return a single value given a single key, we create our ow
 
 Note the **memory** attribute on `adopters`. The memory attribute tells Solidity to temporarily store the value in memory, rather than saving it to the contract's storage. Since `adopters` is an array, and we know from the first adoption test that we adopted pet `expectedPetId`, we compare the testing contracts address with location `expectedPetId` in the array.
 
+## Testing the smart contract using Javascript 
+
+Truffle is very flexible when it comes to smart contract testing, in that tests can be written either in JavaScript or Solidity. In this tutorial, we'll be writing our tests in Javascript using chai and mocha library.
```

Comment:
Same as the comment above about "in this tutorial" being changed to "in this section".